app/controllers/feedbacks_controller.rb

```diff
@@ -0,0 +1,69 @@
+class FeedbacksController < ApplicationController
+  before_action :set_feedback, only: [:show, :edit, :update, :destroy]
+  before_action :set_feedbackable
+  before_action :set_volunteer
+
+  def index
+    authorize Feedback
+    @feedbacks = policy_scope(Feedback)
+    @feedbacks = @feedbacks.where(feedbackable: @feedbackable)
```

Comment:
Doesn't make a difference, but this could be in one line I guess.
`@feedbacks = policy_scope(Feedback).where(feedbackable: @feedbackable)`

Reply:
true. Ill chaange it.